.github/ISSUE_TEMPLATE.md

```diff
@@ -2,10 +2,11 @@ Please refer to the following resources for help using OpenKore:
  faqs - http://openkore.com/index.php/Frequently_Asked_Questions (such as "Packet Tokenizer")
  forum - http://forums.openkore.com
  documentation wiki - http://openkore.com/index.php/Main_Page
-
+ download new openkore - https://github.com/OpenKore/openkore/archive/master.zip
```

Comment:
How about make it large & add like this?
# * Before submitting an issue, ensure you are on the latest release
# * Check if an issue already exists for your request
# * Please post in English if possible
# * Please provide all necessary information or your issue will be CLOSED.

If still no one read it, RIP eyes.